exercises/practice/bob/.approaches/if-statements/content.md

```diff
@@ -0,0 +1,67 @@
+# `if` statements
+
+```javascript
+export function hey(message) {
+  const speech = message.trimEnd();
+  if (speech == '') return 'Fine. Be that way!';
+
+  const isQuestion = speech.endsWith('?');
+  const isShout = /[A-Z]{1}/.test(speech) && speech == speech.toUpperCase();
+
+  if (isShout)
+    return isQuestion
+      ? "Calm down, I know what I'm doing!"
+      : 'Whoa, chill out!';
+  if (isQuestion) return 'Sure.';
```

Comment:
Not using braces `{` for if statements is very non-standard in JavaScript. I see them a lot in the approaches and whilst mostly a stylistic choice, I don't think it's smart to have so many examples with a defacto not-accepted style.

Reply:
Thanks, the javascript is how the formatter formatted it, I believe.

Reply:
`prettier` would never format it like this if brackets were present.

Reply:
So does running format in a comment not actually format the code correctly? If not, sorry, that was what I was depending on.

Reply:
It does!

In generally prettier will not add or remove brackets. It _will_ however format the code inside the brackets, if they are present :)